tarok/src/cards.cpp

```diff
@@ -117,24 +117,11 @@ void Shuffle(std::vector<open_spiel::Action>* actions, std::mt19937&& rng) {
 
 int CardPoints(const std::vector<open_spiel::Action>& actions,
```

Comment:
Maybe we should add a short comment at the top to indicate that subtracting `2/3` from each card is equivalent to counting in threes with a 2-point deduction just for clarity?

Reply:
Added a short description.